README.md

```diff
@@ -91,7 +91,7 @@ final TeamClient teamClient = organisationClient.createTeamClient();
 ## Supported Java versions
 
 This library is written and published with Java version 11. In our CI workflows, we execute
-automated tests with the Java LTS versions 11, 17 and 21. Due to Java's backwards compatability,
+automated tests with the Java LTS versions 11, 17, and 21. Due to Java's backward compatibility,
```

Comment:
```suggestion
automated tests with the Java LTS versions 11, 17 and 21. Due to Java's backward compatibility,
```

I sadly missed noticing this comma. Staying consistent with what I said before, this comma is also not needed.